### DISCUSSION TITLE
Request for a docker image from DockerHub

### DISCUSSION THREAD

**Discussion by xiaohai-huang** [rsxdalv/TTS-WebUI#117] [General]
Can you provide a docker image and upload it to DockerHub? So that I can directly run the webui using just one docker command without having to run `docker build` or install stuff on my computer.

ps: My internet is weird and unstable.

A lot of errors associated with networking.
![image](https://github.com/rsxdalv/tts-generation-webui/assets/56380096/c1e6dcff-187d-4d9d-8207-4768f2ce83c5)

**Comment by rsxdalv** [comment]
It's doable but I have a lot of things happening right now. I'll check if I have the docker credentials set up to make it easier.

**Reply by xiaohai-huang** [reply]
You can use GitHub Actions (CI) to publish docker images automatically.

https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

**Reply by rsxdalv** [reply]
Thank you! I have experience with those so it's a good choice!
May I merge the work from your fork into the main repo?

**Reply by xiaohai-huang** [reply]
> Thank you! I have experience with those so it's a good choice!
> May I merge the work from your fork into the main repo?

Of course. Do I need to create a pull request?

**Reply by rsxdalv** [reply]
Yes, that way you would also be shown as a contributor!
